src/cli/build.ts

```diff
@@ -244,6 +244,10 @@ export async function loadModule(entryPoint = peprTS) {
 export async function buildModule(reloader?: Reloader, entryPoint = peprTS, embed = true) {
   try {
     const { cfg, modulePath, path, uuid } = await loadModule(entryPoint);
+    const excludedDeps = [
+      ...externalLibs,
+      ...Object.keys(cfg.devDependencies).filter(dep => dep !== "typescript"),
```

Comment:
This seems risky to me.

IF esbuild has included a devDependency into the built module it is because a module author has _referenced a dev dependency from their production code_, which means it's _necessary_ to package it.

Excluding the devDeps from the final module output means that such an erroneous module will build successfully but that it'll fail at runtime because we're not packaging the devDeps into the controller image.  This seems _much_ less ideal that just building the mis-configured dep into the module.

At least in that case the module will still work while they work on fixing their deps, no?

Reply:
Which is an anti-pattern to reference devDepencencies in prod code in the first place. It was just a suggestion from UDS Core to exclude devDeps from prod code and I tend to agree.

Since you feel strongly we will just close the PR and open it again if it comes up in the future.